docs/content/sips/001-spin-deploy.md

```diff
@@ -0,0 +1,47 @@
+title = "SIP 001 - Spin Deploy"
+template = "main"
+date = ""
+
+Summary: A Spin CLI command that will deploy a Spin application to the cloud
+
+Owner: michelle@fermyon.com
+
+Created: March 18, 2022
+
+Updated: March 31, 2022
+
+## Background
+
+A Spin application — (currently described as a spin.toml file) — is a collection of one or more components, each invoked as a result of an event generated by a trigger. The Spin CLI currently enables running a Spin application locally via the `spin up` command. The Spin CLI also offers a set of commands enabling users to package and push a Spin application to a [Bindle](https://github.com/deislabs/bindle) registry.
+
+[Hippo](https://github.com/deislabs/hippo) is a self-hosted, open source, Web Assembly PaaS that allows users to deploy and scale Web Assembly applications at scale. Hippo takes a reference to a bindle, deploys it to the cloud and makes it publicly available. Hippo currently only works with Wagi applications and can currently deploy Wagi applications on a Nomad cluster with Traefik for routing. Additionally, Hippo has a concept called “channels” enabling users to track different version categories of an application (ex. alpha, beta, stable, pre-releases, minor versions, etc.).
```

Comment:
What channel will `spin deploy` use? Does that need another option?

Reply:
By default, the channel will be named "spin-deploy" and yes we'll need another option for that. I'll add that to the doc!